torch/utils/data/datapipes/iter/combining.py

```diff
@@ -132,6 +153,15 @@ def __init__(self, datapipe: IterDataPipe, num_instances: int, buffer_size: int
                 "please be aware of OOM at random places",
                 UserWarning
             )
+        if copy is None:
+            self.copy_fn = None
```

Comment:
nit: we could set it to a `_no_op` function that we define, then we will not need the conditional branch when we yield, but I don't feel strongly about this.

```python
def _no_op(x):
    return x 
```

Reply:
I am not sure which one is better in terms of readability and performance, but happy to change.

Reply:
Also not sure if I placed the `_no_op` function at the right place. I can also pull it into the class as a static method or define it locally inside the constructor. Let me know where you want to have it or you can also move it yourself if you like.